Dockerfile

```
@@ -1,26 +1,51 @@
+ARG VERSION=unspecified
+
+FROM python:3.6-slim-buster
+
+ARG VERSION
+
+###
+# For a list of pre-defined annotation keys and value types see:
+# https://github.com/opencontainers/image-spec/blob/master/annotations.md
+#
+# Note: Additional labels are added by the build workflow.
+LABEL org.opencontainers.image.authors="vm-fusion-dev-group@trio.dhs.gov"
+LABEL org.opencontainers.image.vendor="Cybersecurity and Infrastructure Security Agency"
+
 ###
-# Install everything we need
+# Unprivileged user setup variables
 ###
-FROM python:3.6-slim-buster AS install
-LABEL maintainer="jeremy.frasier@trio.dhs.gov"
-LABEL organization="CISA Cyber Assessments"
-LABEL url="https://github.com/cisagov/scanner"
+ARG CISA_UID=2048
+ARG CISA_GID=${CISA_UID}
+ARG CISA_USER="cisa"
+ENV CISA_GROUP=${CISA_USER}
+ENV CISA_HOME="/home/${CISA_USER}"
 
-ENV HOME=/home/scanner
-ENV USER=scanner
+###
+# Upgrade the system
+###
+RUN apt-get update --quiet --quiet \
+    && apt-get upgrade --quiet --quiet
+
+###
+# Create unprivileged user
+###
+RUN groupadd --system --gid ${CISA_GID} ${CISA_GROUP} \
+    && useradd --system --uid ${CISA_UID} --gid ${CISA_GROUP} --comment "${CISA_USER} user" ${CISA_USER}
 
 ###
 # Dependencies
 #
 # We need redis-tools so we can use redis-cli to communicate with
-# redis.
+# redis.  wget is used inside of gather-domains.sh.
 #
 # Install dependencies are only needed for software installation and
 # will be removed at the end of the build process.
 ###
 ENV DEPS \
     bash \
-    redis-tools
+    redis-tools \
+    wget
```

Related to my above suggestion.

```suggestion
    redis-tools
```